csfieldguide/templates/general/index.html

```diff
@@ -17,15 +17,15 @@
         </div>
         <div class="col-md-8 order-md-1 text-center text-md-left pr-md-5">
           <h1 class="mb-3 bd-text-purple-bright">
-            Computer Science Field Guide
+            {% trans 'Computer Science Field Guide' %}
           </h1>
           <p class="lead">
-            An online interactive resource for high school students learning about computer science.
+            {% trans 'An online interactive resource for high school students learning about computer science.' %}
           </p>
           <p>
             <small>
               <a class="text-white" href="http://cs-field-guide.readthedocs.io/en/latest/changelog.html">
-                Currently {{ VERSION_NUMBER }}
+                {{ VERSION_NUMBER }}
```

Comment:
If the wrapping thing I talked about before works you could probably add back in 'Currently' here

Reply:
Yeah it breaks, prints `Currently {{ VERSION_NUMBER }}` instead of the actual number

Reply:
Lets remove this as it is displayed in the footer